gaeguli/gaeguli-internal.h

```diff
@@ -26,37 +26,47 @@
    \"mode\": %" G_GINT32_FORMAT " \
 }"
 
+#define PIPEWIRE_DEVICE_CLASS_STR "Stream/Output/Video"
+
+#define PIPEWIRE_NODE_NAME_STR "node.name"
```

Comment:
If we use `gaeguli.pipeline_id` as I specify it below, we don't need to access `node.name` at all